sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md

```diff
@@ -2987,6 +3007,219 @@ Accept pipeline input: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -AIBuilderModelSelectedSitesList
+
+This parameter allows administrators to pass a list of SharePoint site URLs to modify the AI builder model and [structured and freeform document processing](https://learn.microsoft.com/en-US/microsoft-365/syntex/form-processing-overview) premium feature's selected sites list. By default this parameter overwrites the existing list with the user input list. Additionally, the `AIBuilderModelSelectedSitesListOperation` parameter can be used to specify a different operation. This parameter can only be called if the AI builder model's scope is set to "SelectedSites". The inputted list of site URLs cannot exceed 100 items.
+
+> [!NOTE]
+> Use of this parameter requires that the tenant either have the required license or pay-as-you-go billing set up. For more information, visit [Licensing for Microsoft Syntex](https://learn.microsoft.com/en-us/microsoft-365/syntex/syntex-licensing).
+
+```yaml
+Type: String[]
+Parameter Sets: (All)
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -AIBuilderModelSelectedSitesListOperation
+
+This parameter allows administrators to specify the operation to perform on the AI builder model and [structured and freeform document processing](https://learn.microsoft.com/en-US/microsoft-365/syntex/form-processing-overview) premium feature's current selected sites list using the list of site URLs passed to the `AIBuilderModelSelectedSitesList` parameter.
+
+The valid values are:
+
+- `Overwrite`: Overwrite the existing selected sites list. This is the default operation.
+- `Append`: Append the input list of sites to the existing selected sites list.
+- `Remove`: Remove the input list of sites from the existing selected sites list.
+
+> [!NOTE]
+> Use of this parameter requires that the tenant either have the required license or pay-as-you-go billing set up. For more information, visit [Licensing for Microsoft Syntex](https://learn.microsoft.com/en-us/microsoft-365/syntex/syntex-licensing).
+> Calling this parameter without `AIBuilderModelSelectedSitesList` has no effect.
+
+```yaml
+Type: SelectedSitesListOperations
+Parameter Sets: (All)
+Required: False
+Position: Named
+Default value: Overwrite
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -AIBuilderModelSelectedSitesIncludesContentCenters
+
+This parameter allows administrators to choose whether or not the AI builder model and [structured and freeform document processing](https://learn.microsoft.com/en-US/microsoft-365/syntex/form-processing-overview) premium feature is available on all content center sites when the feature's scope is "SelectedSites" even if they are not explicitly included within the selected sites list. This parameter can only be called if the AI builder model's scope is set to "SelectedSites".
```

Comment:
>"SelectedSites" [](http://example.com/codeflow?start=299&length=15)

for cases like here where you want to call out a param or something like that, use backtick instead of quotes, i.e.

```
`SelectedSites`
```

Reply:
I've replaced the double quotes with backticks